aioauth/collections.py

```diff
@@ -33,8 +34,24 @@ class HTTPHeaderDict(UserDict):
         d['hElLo'] == 'world' # >>> True
     """
 
+    def __init__(self, dict=None, /, **kwargs):
```

Comment:
Positional-only parameters are supported starting from Python 3.8. I'm not sure whether we still need to support Python 3.7, but for now, I would stick to maintaining that compatibility.

Reply:
That was mainly copied from original code from `UserDict`. That's why that signature